### ISSUE TITLE
Introduction / Repository Description

### ISSUE THREAD

**Issue by henricook** [50ButtonsEach/fliclib-linux-dist#16]
Would it be safe to propose adding an introduction to the README along the lines of:

"This project will let you connect Flic buttons to your bluez enabled linux system, it can support X buttons and will allow all actions configurable through [what mechanism?]" ? 

I had to read some way down before I knew what i was looking at! The main win of this project for me is that i'm using flics for things like my lightswitches (instead of a traditional lightswitch). Flic -> LIFX. I have an old android phone in the house atm that will control all the flics but running this SDK on a Raspberry Pi would be vastly preferable, will this project help with that?

Best Regards,

Henri

**Comment by richbeales** [comment]
Yes i'm doing the same thing (flics as lightswitches). This project is what you need.  Bluez is still a bit unstable at present however.

**Comment by fabianbergmark** [comment]
@richbeales try these patches out: http://thread.gmane.org/gmane.linux.bluez.kernel/66537

I have been running them successfully for 2 days!

**Comment by henricook** [comment]
If anyone from Flic is reading this i'd really like some kind of 'flic holding' lightswitch panel for the UK, but i'd like some kind of subtle switch on the side rim or something that'll still let me hard-cut the power to the light if I need to. I get a bit freaked out about not being able to cut the power if I want to (i suppose i've always got the fuse). How are you setting it up @richbeales? I'm likely just going to try and stick a flic over the bottom of the switch (sticking it down) to send the message to people its what they should use instead.

**Comment by richbeales** [comment]
Thats what ive done - the switch stays there, with the flic placed directly
above it on the faceplate.
